AcadeMeData/test_degree.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+from AcadeMeData.models import Degree
+
+
+@pytest.mark.django_db
+class TestDegreeModel:
+    @pytest.fixture
+    def generate_degree(self, degree_id=1, name='History', universities="Ben Gurion University, Reichman University",
+                        description="Learn about historic events and their influences on the world"):
+        degree = Degree(degree_id=degree_id, name=name, universities=universities,
+                        description=description)
+        degree.save()
+        return degree
+
+    def test_create_degree(self):
+        degree = Degree.create_degree(degree_id=1, name="History",
```

Comment:
see you are creating the Degree here. so why do you need the fixture? 

You should call the fixture from test like that test_create_degree(self, generate_degree):

Remove line 16, 
now you have to search for degree in database objects

so you test should be changed.
see this as hoe users tests are going 
user = User.get_user('username')  # search for the user in the db by username
Replace the Degree class and search for the content

https://github.com/redhat-beyond/AcadeMe/blob/main/AcadeMeData/tests.py#L17-L20

Reply:
'create_degree' is the fucntion I'm testing so I can't remove that.
Everything else is done.